src/consumer/index.js

```diff
@@ -39,6 +39,12 @@ module.exports = ({
   let runner = null
   let consumerGroup = null
 
+  if (heartbeatInterval >= sessionTimeout) {
+    throw new KafkaJSNonRetriableError(
+      `Consumer heartbeatInterval (${heartbeatInterval}) must be lower than sessionTimeout (${sessionTimeout})`
```

Comment:
Maybe change the text to be something like: "The heartbeat interval be at least 1/3 of the session timeout"